src/converter.jl

```diff
@@ -0,0 +1,66 @@
+using PyCall
+using AutomotiveSimulator
+using AutomotiveVisualization
+
+function createCurve(segid, laneindex, road_geom, lane)
+    tag = LaneTag(segid, laneindex)
+    curve = nothing
+        if road_geom.type_name == "line"
```

Comment:
Should this be indented? It's indented right now but this "if" statement should be indented back one no?

Reply:
Not sure how this really happened. Definitely no need for the extra indent :-p